README.md

```diff
@@ -1,16 +1,16 @@
-# Operon
+# DBOS SDK
 
-Operon (from [DBOS, Inc.](https://dbos.dev)) is a **Typescript framework built on the database** that helps you develop transactional backend applications.
-Operon fully embraces the power of your application database, helping you write backend applications that are reliable by default.
+The DBOS SDK (from [DBOS, Inc.](https://dbos.dev)) is a **Typescript framework built on the database** that helps you develop transactional backend applications.
```

Comment:
Same here -- Should we say DBOS Typescript SDK is a framework built on the database?